node-sdk-example/test/createSandboxWithCustomPatch.test.js

```diff
@@ -0,0 +1,150 @@
+import {
+    ApiClient,
+    CreateSandboxRequest,
+    CustomPatch,
+    EnvOp,
+    EnvValueFrom,
+    EnvValueFromFork,
+    ForkEndpoint,
+    ForkOf,
+    Image,
+    SandboxCustomizations,
+    SandboxFork,
+    SandboxesApi
+} from '@signadot/signadot-sdk';
+import axios from 'axios';
+import {customAlphabet} from 'nanoid';
+import {expect} from 'chai';
+
+const nanoid = customAlphabet('1234567890abcdef', 5);
+
+let previewURL;
+const CLUSTER_NAME = process.env.SIGNADOT_CLUSTER_NAME;
+const SIGNADOT_ORG = process.env.SIGNADOT_ORG; // Enter your Signadot org name here
+const SIGNADOT_API_KEY = process.env.SIGNADOT_API_KEY; // passed from command line
+const options = {
+    headers: {
+        'signadot-api-key': SIGNADOT_API_KEY
+    }
+};
+
+describe('Create sandbox with custom patch', () => {
```

Comment:
Once you have a description for the previous test, you can copy-paste it here and in the python test as well.